mdpow/workflows/dihedrals.py

```diff
@@ -299,6 +289,33 @@ def get_dihedral_groups(solute, atom_indices):
     return dihedral_groups
 
 def get_paired_indices(atom_indices, bond_indices, dihedral_groups):
+    '''Combines `atom_indices` and `bond_indices` in tuples
+       to be paired with their respective `dihedral_groups` in
+       :func:`~MDAnalysis.core.groups.select_atoms` selection string format.
+
+       A dictionary is created with key-value pairs as follows:
+       `atom_indices` and `bond_indices` are joined in a tuple
+       as the value, with the key being the respective member
+       of `dihedral_groups` to facilitate highlighting the 
+       relevant dihedral atom group when generating violin plots.
+
+       :keywords:
+
+       *atom_indices*
+           tuple of tuples of indices for each dihedral atom group
+
+       *bond_indices*
+           tuple of tuples of indices for the bonds in each dihedral atom group
+
+       *dihedral_groups*
+           list of :func:`numpy.array` for atom names in each dihedral atom group
+
+       :returns:
+
+       *ab_pairs*
+           dictionary with key-value pair: '*#-*#-*#-*#': (atom_indices[i], bond_indices[i])
```

Comment:
what does `*#` mean? `<ATOMNAME><ATOMNUMBER>` ?

Reply:
An example is C1-N2-O3-S4, so I changed it to list that as an example, since it is not technically the atom name or actual index number (the difference between 0 and 1-based atom index and label index is documented elsewhere).